src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -224,20 +226,88 @@ public void AddListener(View targetView, DragAndDropEventHandler callback)
         /// </summary>
         /// <param name="targetView">The target view</param>
         /// <param name="callback">The callback function to remove</param>
-        /// <since_tizen> 10 </since_tizen>
         public void RemoveListener(View targetView, DragAndDropEventHandler callback)
         {
             if (!targetEventDictionary.ContainsKey(targetView))
             {
-                 throw new InvalidOperationException("Fail to RemoveListener");
+                 throw new InvalidOperationException("Fail to RemoveListener for View");
             }
 
             InternalDragAndDropEventHandler cb = targetEventDictionary[targetView];
             targetEventDictionary.Remove(targetView);
             if (!Interop.DragAndDrop.RemoveListener(SwigCPtr, View.getCPtr(targetView),
                                                     new global::System.Runtime.InteropServices.HandleRef(this, Marshal.GetFunctionPointerForDelegate<Delegate>(cb))))
             {
-                 throw new InvalidOperationException("Fail to RemoveListener");
+                 throw new InvalidOperationException("Fail to RemoveListener for View");
+            }
+        }
+
+        /// <summary>
+        /// Adds listener for drop targets
+        /// </summary>
+        /// <param name="targetWindow">The target Window</param>
+        /// <param name="callback">The callback function to get drag event when the drag source enters, moves, leaves and drops on the drop target</param>
+        public void AddListener(Window targetWindow, DragAndDropWindowEventHandler callback)
```

Comment:
Pleass add hidden mark to avoid ACR requiest

Reply:
I added :)